app/models/visitors_step.rb

```diff
@@ -3,8 +3,24 @@ class VisitorsStep
   include Person
 
   attribute :email_address, String
+  attribute :override_spam_or_bounce, Boolean
+  attribute :spam_or_bounce, String
+  attribute :spam_or_bounce_has_occurred, Boolean
```

Comment:
A thought occurs to me somewhat late in the day: perhaps we should call it something like `*delivery_problem*` rather than `*spam_or_bounce*`. These would then be:

* `override_delivery_problem`
* `delivery_problem_type`
* `delivery_problem_occurred`

And I think that would explain itself better.